spyder/utils/programs.py

```diff
@@ -967,6 +981,20 @@ def check_python_help(filename):
         return False
 
 
+def get_python_site_packages(pyexec):
+    """Get the site-packages for a given python executable"""

```

Comment:
Docstrings are sentences so let's add the period :-) 

`"""Get the site-packages for a given python executable."""`